model/src/main/java/com/basistech/rosette/dm/AttributeKey.java

```diff
@@ -52,8 +52,8 @@
     TRANSLATED_TOKENS("translatedTokens"),
     TOPIC_RESULTS("topicResults"),
 
-    CONCEPT("concepts"),
-    KEYPHRASE("keyphrases");
+    CONCEPT("concept"),
```

Comment:
After Hannah's comments, we'll revert this change (ideally we'll want to update `token`, `sentence`, and `scriptRegion` to match all other option's plural naming scheme in the future)